cypress/e2e/pages/articles/testsForLiteOnly.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable import/prefer-default-export */
+import { liteEnabledServices } from '#app/components/LiteSiteCta/liteSiteConfig';
+
+export const testsForLiteOnly = ({ service, pageType }) => {
+  describe(`Running testsForLiteOnly for ${service} ${pageType}`, () => {
+    describe('CTA: Lite', () => {
+      it('should render a call to action component', () => {
+        if (liteEnabledServices.includes(service)) {
+          cy.get('[data-e2e="lite-cta"]').should('be.visible');
```

Comment:
👋 @Isabella-Mitchell and @shayneahchoon, not sure if you saw my comment on the other PR: https://github.com/bbc/simorgh/pull/12074/files#r1810472220

I am wondering if there's an opportunity to write a fast integration test, but also to make the E2E test be more reflective of a user journey (since cypress tests are more expensive, run slower, and should be ideally be more for user interactions)

Reply:
Just looking into this now

Reply:
I've expanded on the e2e test to cover user interactions. Do you think I should remove this 'be.visible' one?

We've also also added an integration test. Currently only one in pages/articles. (Probably ideal to add more over time to check component behaviours).

Reply:
I think it could be removed, since that bit is covered by the integration tests.  The tests below which click on the link & navigate back are much more useful as E2E tests, thanks for adding them!